source/Dgame/Graphic/Color.d

```diff
@@ -29,6 +29,462 @@ import derelict.sdl2.types;
 
 package(Dgame):
 
+static struct Colors(T) {
```

Comment:
Instead of a static struct, you could and should use a mixin template. Then you can also use Color4b.Black etc. at compile time.